Formula/yaml-language-server.rb

```diff
@@ -0,0 +1,114 @@
+require "language/node"
+
+class YamlLanguageServer < Formula
+  desc "Language Server for Yaml Files"
+  homepage "https://github.com/redhat-developer/yaml-language-server"
+  url "https://registry.npmjs.org/yaml-language-server/-/yaml-language-server-1.2.2.tgz"
+  sha256 "c5413137ec0ebf79a4368fd97611f2ae6d08a0c4ee2d6efb28763982a7e05e2a"
+  license "MIT"
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
```

Comment:
Can this test not be run through a port instead of stdin? That would make it a lot easier. If that is not possible, see https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/kotlin-language-server.rb on how to use Homebrew standards for this

Reply:
I have simplified the test. I am not able to use `pipe_output` because of the same reason as #91703:

> Lastly, I tried to write this test using pipe_output as requested, but it does not work with solargraph. The reason is that the language server shuts down before anything is written to stdout. It takes some time for the language server to asynchronously process the initialization request, and output anything to stdout. pipe_output doesn't allow for this so I am still using the Open3.popen3 method as recommended in the documentation. This allows me to call sleep so that the server has time to output the response to stdout.